makefile

```diff
@@ -0,0 +1,27 @@
+# Makefile for Writing Make Files Example
+
+# *****************************************************
+# Variables to control Makefile operation
+
+CXX = g++
+CXXFLAGS = -Wall -g -std=c++11
+LDFLAGS =  /usr/lib/libgtest_main.a -lgtest -lpthread
+# /usr/lib/libgtest.a /usr/lib/libgtest_main.a -lpthread
+
+# ****************************************************
+# Targets needed to bring the executable up to date
+
+main: main.cpp game/yahtzee.h game/die.h
+	$(CXX) $(CXXFLAGS) -o main main.cpp game/yahtzee.cpp game/scoring.cpp game/die.cpp
```

Comment:
This depends on more than it should. It should actually depend on linker files for all the .cpp files.